src/web_interface/rest/rest_binary_search.py

```diff
@@ -74,4 +74,11 @@ def get(self, search_id=None):
         if result is None:
             return error_message('The result is not ready yet or it has already been fetched', self.URL)
 
-        return success_message({'binary_search_results': result}, self.URL)
+        # the "new" binary search result has the structure {<uid>: {<rule>: [<str_match_data>]}}
```

Comment:
Acknowledged. Although we should strongly consider dropping this compatibility #restapiv2

Reply:
We need to add a GraphQL search endpoint anyway, so maybe we can combine this (and maybe also remove the "Advanced Search" endpoint then)